Runtime/GameLiftServerProcessConfiguration.cs

```diff
@@ -1,13 +1,14 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-namespace AmazonGameLift.Runtime
+namespace AmazonGameLiftPlugin.Core
 {
     [System.Serializable]
-    public struct GameLiftConfiguration
+    public struct GameLiftServerProcessConfiguration
     {
         public string AwsRegion;
         public string UserPoolClientId;
         public string ApiGatewayEndpoint;
+        public bool IsGameLiftAnywhere;
```

Comment:
I'm confused by the code that already exists today! Looking at it - There are two struct/classes - GameLiftClientSettings/GameLiftConfiguration. Is GameLiftClientSettings a structure to represent UX attributes and GameLiftConfiguration our internal condensed version of the same? 

If that's the case I'm wondering if there's a way to abstract this class to contain enough info for the managed fleet use case/anywhere use case without having to add this variable in. There could be other variables - but nice to have an abstraction of the configuration which doesn't required details about the mode itself. 

To elaborate the question, I guess how do you intend to use
1.  AwsRegion
2. ApiGatewayEndpoint

for Anyhwere fleets?